### ISSUE TITLE
2.1.2 Remove Duplicates from Sorted Array II

### ISSUE THREAD

**Issue by ByzanTine** [soulmachine/leetcode#56]
Code 2: There isn't any occur variable in this solution. I am confused by what you are talking about.

**Comment by vendisky** [comment]
= =
其实soulmachine同学的意思是，上面的代码1，把所有出现的2，改为3，就能解决最多允许3个重复了

**Comment by ByzanTine** [comment]
Yeah, it certainly make sense, but I think it's better to explicitly declare the occur variable and move this part of comment on code 1. 
Thanks for the clarification anyway.

**Comment by vendisky** [comment]
Exactly. The work of soulmachine is brilliant. Nevertheless, we readers should help him to correct some small mistakes.
